mappings/0a88787e7ce44395085e17a080ed479d86c61267aa280aecfe18176a41646f6e6973.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "0a88787e7ce44395085e17a080ed479d86c61267aa280aecfe18176a41646f6e6973",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://adonistoken.com",
```

Comment:
Hi @mrsqurriel,

policy id is not visible on the website. Can you please update that?

Thanks!

Reply:
The policy id is now visible at the bottom of the main page.
Thanks!